USERS.md

```diff
@@ -321,3 +321,4 @@ Currently, the following organizations are **officially** using Argo CD:
 1. [ZDF](https://www.zdf.de/)
 1. [Zimpler](https://www.zimpler.com/)
 1. [ZOZO](https://corp.zozo.com/)
+1. [Percona](https://percona.com/)
```

Comment:
Could this be moved up in alphabetical order?

Reply:
Of course :)
I am trying to signoff those as well., thank you!